include/qengine_opencl.hpp

```diff
@@ -136,7 +137,8 @@ class QEngineOCL : virtual public QEngine {
 
     QEngineOCL(bitLenInt qBitCount, bitCapInt initState, qrack_rand_gen_ptr rgp = nullptr,
         complex phaseFac = complex(-999.0, -999.0), bool doNorm = false, bool randomGlobalPhase = true,
```

Comment:
Swap out these -999’s as well, if the intent is similar, or create a COMPLEX_UNUSED that can be compared against.

Reply:
Done.